omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/nginx_chef_api_lb.conf.erb

```diff
@@ -71,9 +83,11 @@
     #   /compliance/organizations/ORG/owners/OWNER/compliance[/PROFILE]
     location ~ (?:/compliance)?/organizations/([^/]+)/owners/([^/]+)/compliance(.*) {
       set $request_org $1;
-      set $new_uri "/compliance/profiles/$2$3";
-      access_by_lua_file '/var/opt/opscode/nginx/etc/scripts/check_token_using_endpoint_compliance.lua';
-      proxy_pass $upstream;
+      access_by_lua_block { validator.validate("GET") }
+      proxy_set_header x-data-collector-token $data_collector_token;
```

Comment:
Chef Automate is verifying the data collector token in nginx. Otherwise the response is denied.